modules/oe_content_publication/oe_content_publication.post_update.php

```diff
@@ -29,3 +32,74 @@ function oe_content_publication_post_update_00001_update_field_labels(array &$sa
 function oe_content_publication_post_update_00002() {
   \Drupal::service('module_installer')->install(['oe_content_documents_field']);
 }
+
+/**
+ * Create the new fields in the Publication content type.
+ */
+function oe_content_publication_post_update_00003(): void {
+  $path = drupal_get_path('module', 'oe_content_publication');
+
+  $storage = new FileStorage($path . '/config/post_updates/00003_create_fields');
+  \Drupal::service('config.installer')->installOptionalConfig($storage);
+}
+
+/**
+ * Update field labels.
+ */
+function oe_content_publication_post_update_00004(array &$sandbox): void {
```

Comment:
`$sandbox` isn't used and can be removed.

Reply:
This is fixed.